spec/models/assignment_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'rails_helper'
+
+RSpec.describe Assignment, type: :model do
+  i18n_scope = 'activerecord.errors.models.assignment.attributes'
+  let(:assignment) { FactoryGirl.build(:assignment) }
+
+  it "has a valid factory" do
+    expect(assignment).to be_valid
+  end
+  describe "#attributes" do
+    it "is invalid without an offspring" do
+      assignment.offspring = nil
+      assignment.valid?
+      expect(assignment.errors[:offspring]).to include(I18n.t('offspring.blank', scope: i18n_scope))
+    end
+    it "is invalid without a user" do
```

Comment:
Ok, I was implementing the code not to accept them, and then add the change later on when the different context (user/admin) were more advanced, for clarity. But will change it then